scripts/asm-installer/tests/run_migration_suite_meshca

```diff
@@ -123,6 +127,13 @@ main() {
     set -e
   fi
 
+  echo "Checking for validation service yaml..."
+
+  if [[ ! -f mesh_migration/istiod-service.yaml ]]; then
```

Comment:
np: use variable to represent the filename? Not a must, but depending on how to manage all filenames

Reply:
I always waffle for tests. One one hand changing the filename will require a change, on the other hand if the file accidentally get changed it would be nice for the test to catch it.